src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -91,37 +105,77 @@ function TransactionRequestContent({ request, onConfirm, onReject }: Props) {
   const chainConfig = chainConfigs[chain];
   const value = wrapBigNumberOrNil(transactionPayload?.amount);
   const symbol = transactionPayload?.symbol;
-  const confirmTitle = !hasNotEnoughGas ? t('button.confirm') : t('label.notEnoughGas');
+  const confirmTitle = !hasNotEnoughGas ? t('button.swipeConfirm') : t('label.notEnoughGas');
   const isConfirmDisabled = isEstimating || hasNotEnoughGas || !!errorMessage;
 
   return (
     <>
-      <Text color={chainConfig.color}>
-        {title} {t('label.dotSeparator')} {chainConfig.titleShort}
+      <IconView>
+        <Icon name={chainConfig.iconName} width={64} height={64} style={styles.chainIcon} />
+        <ServiceIcon source={{ uri: iconUrl }} />
+      </IconView>
+
+      <Text color={colors.secondaryText} style={styles.body}>
+        {t('walletConnect.requests.transactionRequest')}
       </Text>
 
-      <Image source={{ uri: iconUrl }} style={styles.icon} />
+      <LargeFiatTokenValueView
+        value={value}
+        assetAddress={gasAddress}
```

Comment:
Using asset address of gas here seems incorrect. You probably should use `transactionPayload?.contractAddress`.

Reply:
Done.